schema/2.0.0/seurat_encoding.md

```diff
@@ -0,0 +1,226 @@
+
+# Schema - Seurat encoding
+
+Contact: acarr@chanzuckerberg.com
+
+Document Status: _In progress_
+
+Schema version: 2.0.0
+
+
+## Overview
+
+All data submitted to the [cellxgene data portal](https://cellxgene.cziscience.com/) is automatically converted to a Seurat V3 object that can be loaded by the R package [Seurat](https://satijalab.org/seurat/). The conversion is done through [sceasy](https://github.com/cellgeni/sceasy).
```

Comment:
Editorial:

All  AnnData datasets submitted to the [cellxgene Data Portal](https://cellxgene.cziscience.com/) are automatically converted to a Seurat V3 object which users can [download](https://docs.cellxgene.cziscience.com/portal/data-portal#download-data) and read into R Packages such as [Seurat](https://satijalab.org/seurat/).

---

RE _The conversion is done through_ that is an implementation detail that could change. It is important to share this with consumers?

Reply:
No I don't believe so. Will remove.